docs/helm-chart/manage-dags-files.rst

```diff
@@ -77,6 +77,12 @@ If you are deploying an image with a constant tag, you need to make sure that th
       --set images.airflow.repository=my-company/airflow \
       --set images.airflow.tag=8a0da78 \
       --set images.airflow.pullPolicy=Always
+      
```

Comment:
I would add a message discouraging use of constant tag and add this description below as an example of problems you might get when you do.

Using constant image tags and re-uploading them is an extremely bad practice for production. you are not supposed to do it because you are completely loosing any traceability of what happened. It's equivalent of randomly changing your code in production server without keeping it in any kind of source repository history. Basically i cannot imagine any scenario where it would be acceptable other than one-man-run-airflow for own personal usage for testing/development.